tests/cli-commands/nodeos-tests.js

```diff
@@ -0,0 +1,328 @@
+            assert(startCommand.template == startDefinition.template);
+            assert(startCommand.description = startDefinition.description);
+            assert(startCommand.options == startDefinition.options);
+            assert(startCommand.subcommands.length == 0);
+        });
+
+        it('Should break when there is already running nodeos', async () => {
+            sinon.stub(nodoesDataManager, "nodeosIsRunning").returns(true);
+
+            assert(await startCommand.execute({ path: nodeosDataPath }));
+
+            sinon.assert.notCalled(pathOptionSpy);
+            sinon.assert.notCalled(setNodeosPathSpy);
+        });
+
+        it('Should start nodeos successfully', async () => {
+            let stub = sinon.stub(nodoesDataManager, "nodeosIsRunning");
+            stub.onFirstCall().returns(false);
+            stub.onSecondCall().returns(true);
+
+            assert(await startCommand.execute({ path: nodeosDataPath }));
+
+            sinon.assert.calledWith(pathOptionSpy, nodeosDataPath);
+            sinon.assert.calledWith(setNodeosPathSpy, nodeosDataPath);
+            sinon.assert.calledWith(templateSpy, nodeosDataPath);
+            sinon.assert.calledOnce(predefinedAccountsSpy);
+        });
+
+        it('Should throw when unexpected error occurred', async () => {
+            let stub = sinon.stub(nodoesDataManager, "nodeosIsRunning");
+            stub.onFirstCall().returns(false);
+            stub.onSecondCall().returns(false);
+
+            assert(await startCommand.execute({ path: nodeosDataPath }));
+
+            sinon.assert.calledWith(pathOptionSpy, nodeosDataPath);
+            sinon.assert.calledWith(setNodeosPathSpy, nodeosDataPath);
+            sinon.assert.calledWith(templateSpy, nodeosDataPath);
+            sinon.assert.notCalled(predefinedAccountsSpy);
+        });
+
+    });
+
+    describe('StopCommand', function () {
+        let stopCommand;
+        let fsReadFileSpy;
+        let fsRemoveDirSpy;
+        let fsRemoveFileSpy;
+
+        beforeEach(async () => {
+            stopCommand = new StopCommand();
+            sinon.stub(AsyncSoftExec.prototype, "exec");
+            sinon.stub(nodoesDataManager, "nodeosPath").returns(nodeosDataPath);
+            fsReadFileSpy = sinon.spy(fileSystemUtil, "readFile");
+            fsRemoveFileSpy = sinon.spy(fileSystemUtil, "rmFile");
+            fsRemoveDirSpy = sinon.spy(fileSystemUtil, "recursivelyDeleteDir");
+        });
+
+        it('Should initialize command properly', async () => {
+            assert(stopCommand instanceof Command);
+            assert(stopCommand.template == stopDefinition.template);
+            assert(stopCommand.description = stopDefinition.description);
+            assert(stopCommand.options == stopDefinition.options);
+            assert(stopCommand.subcommands.length == 0);
+        });
+
+        it('Should clean only the data when there is no running nodeos', async () => {
```

Comment:
Should clean only the data when there is no running nodeos -> Should clean the data only if  there is no running nodeos